packages/localizations/src/el-GR.ts

```diff
@@ -0,0 +1,692 @@
+      detailsLabel: 'Πρέπει να επαληθεύσουμε την ταυτότητά σας πριν επαναφέρουμε τον κωδικό πρόσβασής σας.',
+    },
+    emailCode: {
+      title: 'Ελέγξτε το email σας',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      formTitle: 'Κωδικός επαλήθευσης',
+      formSubtitle: 'Εισαγάγετε τον κωδικό επαλήθευσης που απεστάλη στη διεύθυνση email σας',
+      resendButton: 'Δεν λάβατε κωδικό; Αποστολή ξανά',
+    },
+    emailLink: {
+      title: 'Ελέγξτε το email σας',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      formTitle: 'Σύνδεσμος επαλήθευσης',
+      formSubtitle: 'Χρησιμοποιήστε τον σύνδεσμο επαλήθευσης που απεστάλη στο email σας',
+      resendButton: 'Δεν λάβατε σύνδεσμο; Αποστολή ξανά',
+      unusedTab: {
+        title: 'Μπορείτε να κλείσετε αυτήν την καρτέλα',
+      },
+      verified: {
+        title: 'Επιτυχής σύνδεση',
+        subtitle: 'Θα ανακατευθυνθείτε σύντομα',
+      },
+      verifiedSwitchTab: {
+        subtitle: 'Επιστροφή στην αρχική καρτέλα για να συνεχίσετε',
+        titleNewTab: 'Έχετε συνδεθεί σε άλλη καρτέλα',
+        subtitleNewTab: 'Επιστροφή στη νέα καρτέλα που άνοιξε για να συνεχίσετε',
+      },
+      loading: {
+        title: 'Σύνδεση σε εξέλιξη...',
+        subtitle: 'Θα ανακατευθυνθείτε σύντομα',
+      },
+      failed: {
+        title: 'Αυτός ο σύνδεσμος επαλήθευσης δεν είναι έγκυρος',
+        subtitle: 'Επιστροφή στην αρχική καρτέλα για να συνεχίσετε.',
+      },
+      expired: {
+        title: 'Αυτός ο σύνδεσμος επαλήθευσης έχει λήξει',
+        subtitle: 'Επιστροφή στην αρχική καρτέλα για να συνεχίσετε.',
+      },
+    },
+    phoneCode: { ...commonTexts.signIn.phoneCode },
+    phoneCodeMfa: { ...commonTexts.signIn.phoneCode, subtitle: '' },
+    totpMfa: {
+      title: 'Aυθεντικοποίηση δύο βημάτων',
+      subtitle: '',
+      formTitle: 'Κωδικός επαλήθευσης',
+      formSubtitle: 'Εισαγάγετε τον κωδικό επαλήθευσης που δημιουργήθηκε από την εφαρμογή αυθεντικοποίησης',
+    },
+    backupCodeMfa: {
+      title: 'Εισαγωγή ενός εφεδρικού κωδικού',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      formTitle: '',
+      formSubtitle: '',
+    },
+    alternativeMethods: {
+      title: 'Χρήση μιας άλλης μεθόδου',
+      actionLink: 'Λήψη βοήθειας',
+      blockButton__emailLink: 'Αποστολή συνδέσμου στο {{identifier}}',
+      blockButton__emailCode: 'Αποστολή κωδικού με email στο {{identifier}}',
+      blockButton__phoneCode: 'Αποστολή κωδικού SMS στο {{identifier}}',
+      blockButton__password: 'Σύνδεση με τον κωδικό πρόσβασής σας',
+      blockButton__totp: 'Χρήση της εφαρμογής αυθεντικοποίησης',
+      blockButton__backupCode: 'Χρήση ενός εφεδρικού κωδικού',
+      getHelp: {
+        title: 'Λήψη βοήθειας',
+        content:
+          'Εάν αντιμετωπίζετε δυσκολία στη σύνδεση στον λογαριασμό σας, στείλτε μας email και θα συνεργαστούμε μαζί σας για να αποκαταστήσουμε την πρόσβαση το συντομότερο δυνατόν.',
```

Comment:
@EmmanouelaPothitou "και θα επικοινωνήσουμε...." instead of "και θα συνεργαστούμε...."